File: Lista de Exercícios (C)/C/Lista 1 - Sala/2.c
#include <stdio.h>

int main()
{
    float C, F;
    
    printf("Conversor: Fahrenheit -> Graus Centígrados\n\n");

    printf("--Valores decimais devem ser digitados com (.)!--\n");
    
    printf("Graus (Fahrenheit): "); scanf("%f", &F);
    
    C=(((F-32)*5)/9);
    
    printf("\nGraus (Centígrados): %.1fºC\n", C);
}

File: Listas de Exercícios (Python)/Python/Lista 1 - Sala/2.py
print("Conversor: Fahrenheit -> Graus Centígrados\n")

print("--Valores decimais devem ser digitados com (.)!--")

F=float(input("Graus (Fahrenheit): "))

C=(((F-32)*5)/9); C="{:.1f}".format(C)

print(f"\nGraus (Centígrados): {C}ºC\n")